src/controllers/hid/hidcontroller.cpp

```diff
@@ -67,9 +72,12 @@ int HidController::open() {
 
     // If that fails, try to open device with vendor/product/serial #
     if (!pHidDevice) {
-        qCWarning(m_logBase) << "Failed. Trying to open with make, model & serial no:"
-                             << m_deviceInfo.getVendorId() << m_deviceInfo.getProductId()
-                             << m_deviceInfo.getSerialNumber();
+        qCWarning(m_logBase)
+                << "Failed. Trying to open with make, model & serial no:"
```

Comment:
make, model is hard to understand for me. How about: 
```suggestion
                << "Failed to open with ID (vendor product "serial"):
```